src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,70 @@
-import React from 'react';
+import React, {
+  useState,
+} from 'react';
 import './FindMovie.scss';
+import { MovieCard } from '../MovieCard';
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  addMovie: (movie: Movie) => void;
+};
+
+export const FindMovie: React.FC<Props> = ({ addMovie }) => {
+  const [query, setQuery] = useState('');
+  const [movie, setMovie] = useState<Movie>();
+  const [isLoading, setIsLoading] = useState(true);
+  const [isError, setIsError] = useState(false);
+
+  const findMovie = async (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    const findData = await getMovie(query);
+
+    try {
+      if ('Error' in findData) {
+        setIsError(true);
```

Comment:
If you use a string type you will pass "Can&apos;t find a movie with such a title" as value here